schemas/npi_tables.sql

```diff
@@ -0,0 +1,43 @@
+DROP TABLE IF EXISTS meta.npi_variables;
+CREATE TABLE meta.npi_variables (
+    id serial primary key,
+    name text UNIQUE
+);
+
+COMMENT ON TABLE meta.npi_variables IS E'This table contains a list of variables and their corresponding identifiers used in `data.npi`. The variables include:
+
+* closing_of_public_venues
+* lockdown
+* non-essential_services_closure
+* religious_gatherings_banned
+* shelter_in_place
+* social_distancing
+* gathering_size_10_0
+* gathering_size_25_11
+* gathering_size_100_25
+* gathering_size_500_101
```

Comment:
Should this be `_50_100`?

Reply:
There was a little bit of inconsistency in the naming for gathering sizes (see also the `25_11` and `100_25` categories)

Below is a list of all of the categories.

```python
In [7]: df["npi"].unique()                                                      
Out[7]: 
array(['closing_of_public_venues', 'gathering_size_10_0',
       'gathering_size_100_25', 'gathering_size_25_11',
       'gathering_size_500_101', 'lockdown',
       'non-essential_services_closure', 'religious_gatherings_banned',
       'school closure', 'shelter_in_place', 'social_distancing'],
      dtype=object)
```

Reply:
I'm going to see whether it can be fixed at the raw data level

Reply:
lmk when it is ready for re-review